Xabe.FFmpeg/Conversion/IConversion.cs

```diff
@@ -37,6 +37,22 @@ public interface IConversion
         /// <returns></returns>
         IConversion SetPriority(ProcessPriorityClass? priority);
 
+        /// <summary>
+        /// Extracts Every frameNo frame of the input video and outputs as a png image

```

Comment:
i don't understand  this comment without knowing ffmpeg doc. That i think API doc should be as easy as possible